hack/update-all.sh

```diff
@@ -90,4 +90,9 @@ do
 	fi
 done
 
-echo -e "${color_green}Update scripts completed successfully${color_norm}"
+if kube::util::ensure_clean_working_dir &>/dev/null; then
+    exec "${KUBE_ROOT}/hack/update-all-staging.sh" "$@"
```

Comment:
As a pragmatic solution we could just ask the user: "Hey, something changed. Go to a second terminal and commit. Then continue here."